components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -587,6 +587,11 @@ function updateBeanAndPostTo(postURL, data) {
                            		$('#claim_dialect_wso2').attr('checked', (element.val() == 'custom'));
                            		$('#claim_dialect_custom').attr('checked', (element.val() == 'local'));
                            	});
+                           	jQuery('.ui-dialog-titlebar-close').click(function(){
```

Comment:
Think of a scenario as follows:
1. Click on Use Local Claim Dialect
2. Add a 'Requested Claims' 
3. Click on Define Custom Claim Dialect
4. Close the warning dialog box.
5. According to the existing logic none of the dialect will be selected in this point
Shall we please address that scenario too